itest/lnd_invoice_migration_test.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func openChannelDB(ht *lntest.HarnessTest, hn *node.HarnessNode) *channeldb.DB {
+func openChannelDB(ht *lntest.HarnessTest, hn *node.HarnessNode) kvdb.Backend {
```

Comment:
nit: we should rename this to `openKvBackend` or similar to reflect the new functionality.